clang/include/clang/Basic/Attr.td

```diff
@@ -1197,6 +1197,13 @@ def SYCLDevice : InheritableAttr {
   let Documentation = [SYCLDeviceDocs];
 }
 
+def SYCLGlobalVar : InheritableAttr {
+  let Spellings = [GNU<"sycl_global_var">];
+  let Subjects = SubjectList<[GlobalVar]>;
+  let LangOpts = [SYCLIsDevice];
```

Comment:
A brief comment why this is undocumented would be helpful.

Reply:
Added documentation instead.